SeattleColleges/nsc-events-android
# nsc_events_android-187-creating_an_event

Relates to this user story #186 and #[70](https://github.com/SeattleColleges/nsc-events-nestjs/issues/70) and closes this issue #187 and #[71](https://github.com/SeattleColleges/nsc-events-nestjs/issues/71)

frontend and backend integration. Making a POST call to create an event in the add event page. At the moment, it’s not properly doing any validation checks but that will be the next step.

To test:
- Run the backend locally (it might throw an error if eventCategory in the backend activity create dto is set to _@IsNotEmpty_, so temporarily change that to _@IsOptional_)
- Start up the android app and login from the home page. Navigate to the create event page 
- Fill in all the fields then click “add event” (eventCoverPhoto and eventWebsite must be a url address, eventContact must be an email, eventCapacity must be a number) 
- Check the logcat to see the logging details and/or check mongodb to see the newly created event   

Here's a walkthrough [demo](https://www.youtube.com/watch?v=klmqFgcz_Sw) of the event creation process
